File: Data Structures & Algorithms/concatenation-of-array/submission-2.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    getConcatenation(nums) {
        const n = nums.length;
        const result = new Array(n * 2);

        for(let i = 0; i < n; i++){
            result[i] = nums[i];
            result[i + n] = nums[i];
        }

        return result;
    }
}

File: Data Structures & Algorithms/majority-element/submission-2.cs
public class Solution {
    public int MajorityElement(int[] nums)
    {
        Array.Sort(nums);    
        var result = nums[0];
        var cnt = 0;
        var maxCnt = 0;
        
        for(int i = 0; i < nums.Length - 1; i++)
        {
            if(nums[i] == nums[i + 1])
                cnt++;
            else
            {
                if(cnt > maxCnt)
                {
                    result = nums[i];
                    maxCnt = cnt;
                    cnt = 0;
                }
            }
        }

        return cnt > maxCnt ? nums[nums.Length - 1] : result;
    }
}